### PR TITLE
feat: [ASL-4566] PIL-E MVP iteration 1 - course purpose HE or Training

### PR DESCRIPTION
**Pull request by kiran-varma-home-office**

### CODE REVIEW

**Review by mohomeoffice — APPROVED** [review]
looks okay to me

File: src/model-summary/index.jsx [hunk 1]
```diff
@@ -20,7 +23,9 @@ function ModelProperty({ property, model, formatters, specification, formatNullV
 
     return (
         <Fragment>
-            <dt><Snippet {...snippetProps}>{`fields.${property}.label`}</Snippet></dt>
+            {property === APPLICANT_TRAINING_USE_AT_WORK || property === APPLICANT_LEARNING_USE ?
+                <dt><Snippet {...snippetProps}>{`fields.${property}.checkAnswerLabel`}</Snippet></dt> :
+                <dt><Snippet {...snippetProps}>{`fields.${property}.label`}</Snippet></dt>}
             <dd>
```
Comment (jeff-horton-ho-sas) [inline src/model-summary/index.jsx]: This should not have overrides for specific fields as that is hard to follow. Instead checkAnswerLabel should be a generic property. The Snippet component has a fallback for this use case ([snippet/index.jsx](https://github.com/UKHomeOffice/asl-components/blob/main/src/snippet/index.jsx))

```suggestion
            <dt>
              <Snippet {...snippetProps} fallback={`fields.${property}.label`}>
                {`fields.${property}.checkAnswerLabel`}
              </Snippet>
            </dt> :
            <dd>
```

Comment (kiran-varma-home-office) [reply src/model-summary/index.jsx]: that doesn't work. I only want different label in 2 places and in every other place I want other label. I will call you to discuss!

Comment (kiran-varma-home-office) [reply src/model-summary/index.jsx]: thanks for your help @jeff-horton-ho-sas .

**Review by jeff-horton-ho-sas — APPROVED** [review]
